src/backend/drm/device/mod.rs

```diff
@@ -143,6 +146,13 @@ impl<A: AsRawFd + 'static> DrmDevice<A> {
             .get_driver_capability(DriverCapability::MonotonicTimestamp)
             .unwrap_or(0)
             == 1;
+        let cursor_width = dev
+            .get_driver_capability(DriverCapability::CursorWidth)
+            .unwrap_or(64);
+        let cursor_height = dev
+            .get_driver_capability(DriverCapability::CursorHeight)
+            .unwrap_or(64);
+        let cursor_size = Size::from((cursor_width as u32, cursor_height as u32));
```

Comment:
Shouldn't it fallback to software cursor if the w/h can't be found?

Reply:
This change could be moved to #760 as it is not used in this PR.
It only tells you the size of the hardware cursor, actually using a hardware cursor is handled in #760 (but only for atomic)